build/project-template-gradle/build.gradle

```diff
@@ -270,6 +270,17 @@ copyMetadata.dependsOn(buildMetadata)
 
 task buildapk {
 	
+	  task includePluginBuildFiles {
+
+      new File("$nodeModulesDir").eachFileRecurse(groovy.io.FileType.FILES) {
```

Comment:
I think eachFileMatch is not recursive. 
Other option is to use fileTree for this.